fhir-config/src/main/java/com/ibm/fhir/config/FHIRConfigHelper.java

```diff
@@ -151,5 +155,40 @@ private static JsonValue getPropertyFromTenantOrDefault(String propertyName) {
         }
 
         return (result != null ? result : defaultValue);
+    }    
+    
+    /**
+     * This method returns the list of supported resource types
+     * @return a list of resource types that isn't null
+     */
+    public static List<String> getSupportedResourceTypes() throws FHIRException {
+        List<String> result = new ArrayList<>();
+
+        PropertyGroup rsrcsGroup = FHIRConfigHelper.getPropertyGroup(FHIRConfiguration.PROPERTY_RESOURCES);
+        List<PropertyEntry> rsrcsEntries;
+        try {
+            rsrcsEntries = rsrcsGroup.getProperties();
+
+            if (rsrcsEntries != null && !rsrcsEntries.isEmpty()) {
+                for (PropertyEntry rsrcsEntry : rsrcsEntries) {
+                    String name = rsrcsEntry.getName();
+                    
+                    // Ensure we skip over the special property "open" and process only the others
+                    // and skip the abstract types Resource and DomainResource
+                    // It would be nice to be able to verify if the resource names were valid, but
+                    // not possible at this layer of the code.  
+                    if (!FHIRConfiguration.PROPERTY_FIELD_RESOURCES_OPEN.equals(name) &&
```

Comment:
I wonder if it makes sense to look for an `interactions` config block and ensure that not only is the endpoint supported but that we support `search` on this endpoint

Reply:
actually, i see this is actually a utility, so probably doesn't make sense to do it at this level...will see if it makes sense when i get to where this is called from